File: python/towerOfHanoi.py
def Hanoi(n,src,help,dest):
    if n==1:
        print(src,'->',dest)
        return
    Hanoi(n-1,src,dest,help)
    print(src,'->',dest)
    Hanoi(n-1,help,src,dest)

if __name__== "__main__":
    n=4
    Hanoi(n,'A','B','C')

File: New folder/cpp/towerOfHanoi.cpp
#include <iostream>
#define space ' '
#define newline endl
using namespace std;

void Hanoi(int n, char src,char help,char dest)
{
    if (n == 1){
		cout << src << " -> " << dest << newline;
		return;
	}
	Hanoi(n - 1,src,dest,help);
	cout << src << " -> " << dest << newline;
	Hanoi(n - 1,help,src,dest);
}

int main()
{
	int n = 3; 
	Hanoi(n, 'A', 'B', 'C');
	return 0;
}
